content/sensu-go/6.0/sensuctl/back-up-recover.md

```diff
@@ -0,0 +1,183 @@
+---
+title: "Back up and recover resources with sensuctl"
+linkTitle: "Back Up and Recover Resources"
+description: "Use the sensuctl command line tool to back up and recover your Sensu resources. Read this page to learn how to use sensuctl dump commands for backup and recovery."
+weight: 20
+version: "6.0"
+product: "Sensu Go"
+platformContent: false 
+menu:
+  sensu-go-6.0:
+    parent: sensuctl
+---
+
+The `sensuctl dump` command allows you to export your resources to standard out (STDOUT) or to a file.
+You can export all of your resources or a subset of them based on a list of resource types.
+The `dump` command supports exporting in `wrapped-json` and `yaml`.
+
+{{% notice note %}}
+**NOTE**: Passwords are not included when exporting users.
+You must add the `password` attribute to any exported user resources before you can use them with `sensuctl create`.
+{{% /notice %}}
+
+For example, to export all resources to a file named `my-resources.yaml` in `yaml` format:
+
+{{< code shell >}}
+sensuctl dump all --format yaml --file my-resources.yaml
+{{< /code >}}
+
+To export only checks to STDOUT in `yaml` format:
+
+{{< code shell >}}
+sensuctl dump check --format yaml
+{{< /code >}}
+
+To export only handlers and filters to a file named `my-handlers-and-filters.yaml` in `yaml` format:
+
+{{< code shell >}}
+sensuctl dump handler,filter --format yaml --file my-handlers-and-filters.yaml
+{{< /code >}}
+
+## Back up your resources
+
+Use `sensuctl dump` to back up your Sensu resources so you can [restore][3] them later with [`sensuctl create`][1].
+For example, we recommend creating a backup before you [upgrade][4] to a new version of Sensu.
+
+{{% notice note %}}
+**NOTE**: You cannot export `secrets/v1.provider` resources &mdash; make sure to omit them from your `sensuctl dump` commands.
```

Comment:
I can't reproduce the issue described in https://github.com/sensu/sensu-enterprise-go/issues/1193 anymore, so we should remove this note for now.